web/test/e2e/pages/oppija/KoskiTpoOppijaPage.ts

```diff
@@ -84,6 +84,22 @@ export class KoskiTpoOppijaPage extends KoskiOppijaPageV2<
       .arvosana.set(arvosana)
   }
 
+  async setTunnustettu(selite: string) {
+    await this.$.suoritukset(this.suoritusIndex)
+      .osasuoritukset(this.osasuoritusIndex)
+      .expand.click()
+    await this.page
+      .getByTestId(
+        `suoritukset.${this.suoritusIndex}.osasuoritukset.${this.osasuoritusIndex}.properties.tunnustettu.edit.add`
```

Comment:
Älä rakenna id:tä käsin, vaan käytä tämän saman tiedoston pohjalta löytyvää `TaiteenPerusopetusTestIds` -rakennetta apuna. Se on mäpätty propertyyn `this.$`, kuten tuossa yläpuolella on tehty.

Eli tää olis jotain tyyliin: `this.$.suoritukset(this.suoritusIndex).osasuoritukset(this.osasuoritusIndex).properties.tunnustettu.set(selite)`

Reply:
Tuo set yms. funktiot löytyvät tiedostosta web/test/e2e/pages/oppija/uiV2builder/Input.ts

Reply:
Joo, tota yritin aluks mutta lentää `Error: Trying to set an editor value without editor` ja sitten jumiuduin tohon editorin avaamiseen. `this.$.suoritukset(this.suoritusIndex).osasuoritukset(this.osasuoritusIndex).properties.tunnustettu.edit` on undefined

Reply:
Ilmeisesti ko. inputille ei oo tukea vielä

Reply:
Muuta TaiteenPerusopetusTestIds: `tunnustettu: FormField(Label)` --> `FormField(Label, Input)`

Reply:
Piti tehdä kustomina kun ei toi field oo näkyvissä ennen nappulan painamista